recipes/discount/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class DiscountConan(ConanFile):
+    name = "discount"
+    description = "DISCOUNT is a implementation of John Gruber & Aaron Swartz's Markdown markup language."
+    license = "BSD-3-Clause"
+    topics = ("discount", "markdown")
+    homepage = "http://www.pell.portland.or.us/~orc/Code/discount"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["DISCOUNT_MAKE_INSTALL"] = True
+        self._cmake.definitions["DISCOUNT_INSTALL_SAMPLES"] = False
```

Comment:
By the way, it's off by default.

Reply:
I know, but I like to be explicit (upstream could decide to turn this option to ON by default).